tests/unit_tests/test_wave_01.py

```diff
@@ -40,16 +40,20 @@ def test_removing_from_inventory_returns_item():
     assert item not in vendor.inventory
     assert result == item
 
-@pytest.mark.skip
+# @pytest.mark.skip
 def test_removing_not_found_is_false():
     item = "item to remove"
     vendor = Vendor(
         inventory=["a", "b", "c"]
     )
 
     result = vendor.remove(item)
+    # Assert
+    assert len(vendor.inventory) == 3 
+    assert item not in vendor.inventory 
+    assert result is False
```

Comment:
Excellent! The second assert statement is not necessary since the inventory never had the item to begin it.